homeassistant/components/philips_js/diagnostics.py

```diff
@@ -21,10 +29,7 @@ async def async_get_config_entry_diagnostics(
     api = coordinator.api
 
     return {
-        "entry": {
-            "title": entry.title,
-            "config": async_redact_data(entry.as_dict(), TO_REDACT)
-        },
+        "entry": async_redact_data(entry.as_dict(), TO_REDACT),
```

Comment:
This will also include the unique ID. Not sure if that's the device ID.

Reply:
Indeed. will block too.